source/hash_table.c

```diff
@@ -222,7 +222,7 @@ int aws_hash_table_init(
     AWS_PRECONDITION(alloc != NULL);
     AWS_PRECONDITION(hash_fn != NULL);
     AWS_PRECONDITION(equals_fn != NULL);
-
+    map->allocator = alloc;
```

Comment:
You're storing the allocator here, but surely it needs to be used whenever the table is re-sized right?

Reply:
Yeah, I was wanting to expose the allocator as what we did for other structures(like aws_byte_buffer). Now I see it's awkward to expose this as we have the underlying structure... Will remove.eiifccugt